torch/csrc/jit/ir.cpp

```diff
@@ -584,7 +584,13 @@ void Block::reIndexTopology() {
   }
 }
 
-void Block::cloneFrom(Block* src, std::function<Value*(Value*)> value_map) {
+
+void Block::cloneFrom(Block* src, std::function<Value*(Value*)> value_map)
+{
+  cloneWithProfile(src, value_map, false);
+}
+
+void Block::cloneWithProfile(Block* src, std::function<Value*(Value*)> value_map, bool profile) {
```

Comment:
should this also copy the profiling count?

Reply:
This does: https://github.com/pytorch/pytorch/pull/18772/files#diff-07e06aa4afc5f889bb134d6d6415ad2aR658
Frankly, I'm not very excited about copying and carrying around a profiling count, but it might be useful to know that this graph is being profiled.